Lib/idlelib/config.py

```diff
@@ -345,11 +349,11 @@ def GetThemeDict(self, type, themeName):
                 'stderr-background':'#ffffff',
                 'console-foreground':'#000000',
                 'console-background':'#ffffff',
-                'context-foreground':'#000000',
-                'context-background':'#ffffff',
                 }
         for element in theme:
-            if not cfgParser.has_option(themeName, element):
+            if (not cfgParser.has_option(themeName, element) and
+                # Skip warning for new elements.
+                element[:element.index('-')] not in ['context', 'sidebar']):
```

Comment:
```suggestion
                not element.startswith(('context-', 'sidebar-')):
```

Reply:
I agree, but cannot commit thru this interface.  Says "You do not have permission to push to this repository.", which is wrong for this branch.  Can you accept your own suggestion?